tests/tests_gapfiller/tests_gapfiller.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
```

Comment:
if you start raising user defined exceptions then we can test for exceptions in testing:
https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception